applications/CompressiblePotentialFlowApplication/custom_conditions/potential_wall_condition.cpp

```diff
@@ -298,7 +298,7 @@ void PotentialWallCondition<TDim, TNumNodes>::FindParentElement(
         if (std::includes(ElementNodeIds.begin(), ElementNodeIds.end(),
                           NodeIds.begin(), NodeIds.end()))
         {
-            mpElement = ElementCandidates(i);
+            mpElement = ElementCandidates(i).get();
```

Comment:
Using get here to retrieve the raw pointer (note that mpElement is now a raw pointer, Element*)

Reply:
ElementCandidates is a GlobalPointers vector that comes from the NEIGHBOUR_ELEMENTS variable